images/Dockerfile

```diff
@@ -5,7 +5,7 @@ ARG TARGETOS
 ARG TARGETARCH
 ARG RUNNER_VERSION
 ARG RUNNER_CONTAINER_HOOKS_VERSION=0.5.1
-ARG DOCKER_VERSION=25.0.2
+ARG DOCKER_VERSION=25.0.3
```

Comment:
This is just a bump, and can be excluded. Without this bump the vulnerability is not reported, but I thought why not use the latest patch version when we are here :relaxed: